tests/test_overdrive.py

```diff
@@ -65,7 +65,7 @@ def test_default_notification_email_address(self):
             "patron_info.json"
         )
         api = DummyOverdriveAPI(self._db)
-        api.queue_response(content=patron_with_email)
+        api.queue_response(200, content=patron_with_email)
```

Comment:
The core `MockOverdriveAPI` implementation of `queue_response` has a non-optional argument for status code.